
Fix the last missing bit to fully compile current 2.4 w/ gcc-3.4.x
on ia-32.

  - Rene Rebe <rene@exactcode.de>

--- ./drivers/scsi/ips.h.orig	2004-09-22 01:53:33.000000000 +0200
+++ ./drivers/scsi/ips.h	2004-09-22 01:53:21.000000000 +0200
@@ -97,10 +97,6 @@
 
    #if LINUX_VERSION_CODE < KERNEL_VERSION(2,5,0)
    
-      #ifndef irqreturn_t
-         typedef void irqreturn_t;
-      #endif 
-      
       #define IRQ_NONE
       #define IRQ_HANDLED
       #define IRQ_RETVAL(x)
